docs/index.rst

```diff
@@ -6,54 +6,76 @@ provides a simple interface for performing aperture photometry of either a singl
 image or a directory with multiple images. It is designed to be easy to use for both
 non-programmers and programmers.
 
-
-Getting Started
-===============
-
 Installation
-------------
+============
+
 
 Testing
-+++++++
+-------
 
 If you are testing a pre-release version of stellarphot we recommend setting up
 a virtual environment and installing stellarphot in this environment.
 
-Creating an environment with `conda` or `mamba`:
+Only use one of the methods below for making a virtual environment.
 
-```bash
-mamba create -n stellarphot-test python=3.11
-mamba activate stellarphot-test
-pip install --pre --upgrade stellarphot
-```
+Creating an environment with `conda` or `mamba`::
 
-Creating an environment with `virtualenv`:
+    mamba create -n stellarphot-test python=3.11
+    mamba activate stellarphot-test
+    pip install git+https://github.com/mwcraig/stellarphot.git@update-docs
 
-```bash
-python3 -m venv stellarphot-test
-source stellarphot-test/bin/activate
-pip install --pre --upgrade stellarphot
-```
+Creating an environment with `virtualenv`::
+
+
+    python3 -m venv stellarphot-test
+    source stellarphot-test/bin/activate
+    pip install --pre --upgrade stellarphot
+
+
+Getting Started
+===============
 
 Overview
 --------
 
+You will go through this process to do photometry.
+
+#. You need to make some settings, like camera properties, observatory information, and passband maps. You may only need
+   to do this step once if you use the same equipment for all of your observations.
+#. Settings specific to an object need to be made:
+
+   #. night of data, like the photometry aperture radius, need to be made.
+   #. a list of the sources for which you want to perform photometry. These lists can be re-used.
+
+#. Review all of the settings that the photometry routines will use.
+#. Once those settings have been done, you can perform photometry on your images.
```

Comment:
The instructions here are malformed and say "Choose any image in the folder of images to do photometry on that contains the object of interest".  Can you clarify?  I assume based on the behavior that clicking on an images selects processing of all images containing that object in the folder...

Reply:
I second that we may want to clarify the wording.